engine/installation/index.md

```diff
@@ -53,6 +53,22 @@ platform.
 See also [Docker Cloud](#on-docker-cloud) for setup instructions for
 Digital Ocean, Packet, SoftLink, or Bring Your Own Cloud.
 
+## Time-based release schedule
+
+Starting with Docker 17.03, Docker uses semantic versioning and a time-based
+release schedule, outlined below. There may be times when exceptions need to be
+made, but this is the goal.
+
+- Ideally, a Docker CE release is published the first week of each month.
```

Comment:
@andrewhsu specifically asked me to outline this. I have reservations about it too. I wonder who else should weigh in.